unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -71,6 +76,13 @@
  * @author Sébastien Paumier
  */
 public class Config {
+	
+	private static final int PORT = 9999;
```

Comment:
I had completed latest change and was testing when I received your suggestion so I've submitted. Change does not use jUnique but uses system-generated port saving to local file. I moved single instance logic from Config.java to new class SingleInstanceMonitor. I can change it again if you would prefer to use configuration properties.